src/I18n/DateFormatTrait.php

```diff
@@ -64,9 +74,57 @@
     protected static $_isDateInstance;
 
     /**
+     * Gets the default output timezone.
+     *
+     * @return \DateTimeZone|null DateTimeZone object in which the date will be displayed or null.
+     */
+    public static function getDefaultOutputTimezone()
+    {
+        return static::$_defaultOutputTimezone;
+    }
+
+    /**
+     * Sets the default output timezone.
+     *
+     * @param string|\DateTimeZone $timezone Timezone string or DateTimeZone object
+     * in which the date will be displayed.
+     * @return void
+     */
+    public static function setDefaultOutputTimezone($timezone)
+    {
+        if ($timezone === null) {
```

Comment:
Accepting null does not make much sense in this method. You can remove this case